cmd/evergreen/evergreen.go

```diff
@@ -5,14 +5,16 @@ import (
 	"path/filepath"
 	"runtime"
 
+	// this *must* be included in the binary so that the legacy
```

Comment:
I realize you had to revert this commit to be able to deploy, but can you un-revert in the pr?